Examples/Sudoku-examples/CLIPS-puzzles/grid3x3-p1.clp

```diff
@@ -30,12 +30,18 @@
 ;;; 
 ;;;    Naked Single
 
-
+(bind ?override "(progn
+(bind ?*Subsets* TRUE)
+(bind ?*FinnedFish* TRUE)
```

Comment:
Note that when an override is defined, the entire user configuration is replaced.  Thus, this must contain all options needed for the test.  This enables tests to precisely control the configuration.